HVZ.Persistence.MongoDB/Repos/GameRepo.cs

```diff
@@ -225,6 +232,109 @@ public async Task<Game> LogTag(string gameId, string taggerUserId, string tagRec
         return newGame;
     }
 
+    public async Task<Game> AddPlayerToOzPool(string gameId, string playerId)
+    {
+        Game game = await GetGameById(gameId);
+        Player? player = await FindPlayerByUserId(gameId, playerId);
+        if (player is null)
+        {
+            throw new ArgumentException($"Could not find player with GameId {playerId} in Game {gameId}");
+        }
```

Comment:
```suggestion
        Player player = await GetPlayerByUserId(gameId, playerId);
```
Get... methods have this check built in

Reply:
Get... methods do not currently exist #128